Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/IntakeEject.java

```diff
@@ -133,6 +168,8 @@ public void closeIntake() {
 		DoubleSolenoid.Value rightSet = Robot.getBool("Intake Right Horizontal Solenoid Inverted", false)
 				? DoubleSolenoid.Value.kReverse
 				: DoubleSolenoid.Value.kForward;
+		SmartDashboard.putBoolean("Left Horizontal Solenoid Open", false);
+		SmartDashboard.putBoolean("Right Horizontal Solenoid Open", false);
```

Comment:
The above lines (or ones like them) are repeated many times in this class. Consider refactoring around a new  setIntakeState(IntakeSide side, IntakeState state) method where IntakeSide would be an enum of LEFT and RIGHT, and IntakeState would be an enum of OPEN and CLOSED. setIntakeState() would determine invertedness, update the SmartDashboard, and update the state maintained by the class.

Reply:
Addressed in latest commit.